### DISCUSSION TITLE
Changelog order

### DISCUSSION THREAD

**Discussion by auronandace** [GyulyVGC/sniffnet#5] [Ideas]
I've noticed most projects changelogs have the newest releases at the top and the oldest at the bottom. This can be pretty handy since when you open the page you are at the top and you'll see the latest release details rather than needing to scroll down.

Is there any reason why the changelog is oldest at the top?

**Comment by GyulyVGC** [comment]
Actually there is not a specific reason, I think I will follow your suggestion! Thanks!
